src/elements/image/ImageElement.tsx

```diff
@@ -61,9 +61,10 @@ export const createImageFields = ({
 }: MainImageProps) => {
   return {
     alt: createTextField({
-      rows: 2,
+      rows: 4,
```

Comment:
Increasing the default a result of 
`something resizeable, we're likely expecting lots of content in a limited space`?

Reply:
Yeah, precisely – but maybe it's now a little big?

Reply:
I think we could go for 3 as default middle ground :)
Alternatively I don't think it makes a huge difference IMO

Reply:
Tested inn Composer, 2 is better – thanks for mentioning 👍